client/src/components/PastaTile.js

```diff
@@ -1,7 +1,10 @@
 import React from "react";
+import { Link } from 'react-router-dom'
 
-const PastaTile = ({ name }) => {
-  return <div className="pasta-tile">{name}</div>;
+const PastaTile = (pasta) => {
+  return <div className="pasta-tile">
```

Comment:
Let's start this `div` on a new line

If we were returning one line of code we could write it inline with the return, but the return value is spanning multiple lines, meaning we should open parenthesis
```
return (
  <div>
    ...
  </div>
)
```